src/mentionui.js

```diff
@@ -469,9 +469,11 @@ function createTextMatcher( marker ) {
 // Default feed callback
 function createFeedCallback( feedItems ) {
 	return feedText => {
-		const filteredItems = feedItems.filter( item => {
-			return item.toLowerCase().includes( feedText.toLowerCase() );
-		} );
+		const filteredItems = feedItems
+		// Make default mention feed case-insensitive.
+			.filter( item => item.toLowerCase().includes( feedText.toLowerCase() ) )
+			// Do not return more then 10 items.
+			.slice( 0, 10 );
```

Comment:
Out of curiosity, if anyone wants, let's say 15, can they have that?

Reply:
Using custom callback.

Reply:
We do not filter anything if developer is using custom callback.

Reply:
Cool then.